README.md

```
@@ -1,33 +1,34 @@
+To install Rust, follow the instructions on the [official website](https://www.rust-lang.org/tools/install).
+The installation includes the Rust compiler, Cargo (the Rust package manager) and the Rust standard library.
 
-### Zero-cost Abstractions
+### Building from Source 
 
-...
+To build the project from source, clone the repository and execute the following command in the root directory: 
+```bash 
+cargo build --release 
+``` 
+You can also generate the documentation by running: 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)